wordpress-mobile/WordPress-Android
# Add Gutenberg Embed WebView Activity

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/5509

Adds a new embed WebView for the editor.

Related PRs:
- https://github.com/wordpress-mobile/gutenberg-mobile/pull/5743
- https://github.com/WordPress/gutenberg/pull/50440
- https://github.com/Automattic/jetpack/pull/30521

To test:
1. Apply both related PRs.
2. Open a post with a VideoPress block in the Editor
3. Select the VideoPress block and press again to play the preview video
4. The VideoPress embed player should open in a WebView
5. Exit the WebView via swipe or by pressing the back arrow. 
6. Expect to return to the editor.

## Regression Notes
1. Potential unintended areas of impact
Gutenberg bridge code - minimal.


7. What I did to test those areas of impact (or what existing automated tests I relied on)
Existing code was not altered for these changes. 

8. What automated tests I added (or what prevented me from doing so)

PR submission checklist:


UI Changes testing checklist:
